inst/builtin/bs5/shiny/_rules.scss

```diff
@@ -91,19 +91,30 @@ $icon-classes: ("bi", "fa", "fas", "far", "fab", "material-icons") !default;
   }
 }
 
+
 .bslib-value-box {
-  &.text-info,
-  &.text-cyan {
-    color: #028097 !important; // Contrast 4.63
+  $cyan: #028097; // Contrast 4.63
```

Comment:
oh my gosh... Listing the contrast in the comments!! How has that never occurred to me...  Love it.